onmt/modules/lora.py

```diff
@@ -195,13 +195,18 @@ def train(self, mode: bool = True):
                     pass
 
             def forward(self, x: torch.Tensor):
-                result = self.maybe_ckpt(super().forward, x)
                 if self.r > 0 and not self.merged:
-                    result += (
-                        self.lora_dropout(x)
-                        @ self.lora_A.transpose(0, 1)
-                        @ self.lora_B.transpose(0, 1)
-                    ) * self.scaling
+                    result = (
```

Comment:
in some instance I am getting an inplace backward error when using results += ...